tests/unit/type/zset.tcl

```diff
@@ -2065,6 +2065,32 @@ start_server {tags {"zset"}} {
         close_replication_stream $repl
     } {} {needs:repl}
 
+    test "BZMPOP should not blocks on non key arguments - #10762" {
+        set rd1 [redis_deferring_client]
+        set rd2 [redis_deferring_client]
+        r del myzset myzset2 myzset3
+
+        $rd1 bzmpop 0 1 myzset min count 10
+        wait_for_blocked_clients_count 1
+        $rd2 bzmpop 0 2 myzset2 myzset3 max count 10
+        wait_for_blocked_clients_count 2
+
+        r zadd 0 100 a ;# timeout value
+        r zadd 1 200 b ;# numkeys value
+        r zadd min 300 c ;# min token
+        r zadd max 400 d ;# max token
+        r zadd count 500 e ;# count token
+        r zadd 10 200 b ;# count value
+
+        r zadd myzset 600 f
+        r zadd myzset3 600 f
+        assert_equal {myzset {{f 600}}} [$rd1 read]
+        assert_equal {myzset3 {{f 600}}} [$rd2 read]
+
+        $rd1 close
+        $rd2 close
+    } {0} {external:skip}
```

Comment:
Why external skip? Is it because of the crossslot cluster concern? 
In that case there's a cluster skip tag for that.

Reply:
yes, i didn't notice there was a cluster:skip before, and i see it now in tests/README.md

and find a typo...  i will leave it for further PR (collecting some typo)
| `tls:skip`                | Not campatible with `--tls`. |